hack/test-build.sh

```diff
@@ -28,6 +28,10 @@ if [ -z "$MY_QUAY_USER" ]; then
 else
   if oc get secret redhat-appstudio-staginguser-pull-secret &>/dev/null; then
      PUSH_WORKSPACE="-w name=registry-auth,secret=redhat-appstudio-staginguser-pull-secret"
+     # Ensure that the pipeline service account has access to the secret. Although the
+     # secret is mounted directly on the pipeline, Tekton Chains needs this linkage so
+     # it can push the image signature and attestation to the same OCI repository.
+     oc secrets link pipeline redhat-appstudio-staginguser-pull-secret --for=pull
   else
      echo redhat-appstudio-staginguser-pull-secret is not created, can be created by:
      echo oc create secret docker-registry redhat-appstudio-staginguser-pull-secret --from-file=.dockerconfigjson=$HOME/.docker/config.json
```

Comment:
After this should the link command be printed?

Reply:
I didn't add because I had assumed that if someone needs to create the secret, they'll do that then re-run the script. But it's probably a good idea to be more explicit. Done.